sphinx/application/sphinx/build.gradle

```diff
@@ -116,6 +116,8 @@ dependencies {
     }
 
     implementation deps.google.hilt
+    implementation project(path: ':sphinx:application:network:concepts:concept-meme-input-stream')
```

Comment:
remove. concept module should be provided from the feature module via use of `api` dependency